### PR TITLE
Feature/cha8 login

### PR DESCRIPTION
**Pull request by sswrk**
implemented JWT obtaining and logout

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,6 +1,8 @@
 asgiref==3.3.1
 Django==3.1.7
+django-filter
```
Comment (tmek1244) [inline requirements.txt]: can you add version here?

File: setup.cfg [hunk 2]
```diff
@@ -12,6 +12,7 @@ disallow_incomplete_defs = True
 disallow_untyped_calls = True
 disallow_untyped_decorators = True
 disallow_untyped_defs = True
+ignore_missing_imports = True
```
Comment (tmek1244) [inline setup.cfg:17]: That's not good idea, to ignore every possible missing imports in the future. Change that to only ignore that one library 
`[mypy-rest_framework_simplejwt.*]
ignore_missing_imports = True`

File: users/serializers.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
+from typing import Any, List
+
+from rest_framework_simplejwt.serializers import TokenObtainPairSerializer
+from rest_framework_simplejwt.state import User
+from rest_framework_simplejwt.tokens import RefreshToken
+
+
+class CustomTokenObtainPairSerializer(TokenObtainPairSerializer):
+
+    def validate(self, attrs: List[str]) -> Any:
```
Comment (tmek1244) [inline users/serializers.py:10]: Any will work, but it looks like data is some kind of dictionary

Comment (sswrk) [reply users/serializers.py:10]: I tried that but mypy sees the return type as Any 
image

**Review by skupien — APPROVED** [review]

**Pull request merged**